VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -70,14 +76,31 @@ public void getItemOffsets(Rect outRect, View view, RecyclerView parent, Recycle
         });
         discussionCommentsListView.setAdapter(discussionCommentsAdapter);
 
-        createNewCommentTextView.setText(context.getString(R.string.discussion_post_create_new_comment));
-
-        createNewCommentLayout.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                router.showCourseDiscussionAddComment(context, discussionComment);
-            }
-        });
+        if (isTopicClosed) {
+            createNewCommentTextView.setText(context.getString(R.string.discussion_add_comment_disabled_title));
```

Comment:
You don't need the `context.getString()` here; you can just pass the string ID directly to `getText()`.